sql/patch-schema-2.sql

```diff
@@ -3,3 +3,13 @@ USE ifdb;
 -- use this script for pending changes to the production DB schema
 
 
+CREATE TABLE `tagsynonyms` (
+    `tagsynonymid` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+    `fromtag` varchar(255) COLLATE latin1_german2_ci NOT NULL,
+    `totag` varchar(255) COLLATE latin1_german2_ci NOT NULL,
+    PRIMARY KEY (`tagsynonymid`),
+    KEY `fromtag` (`fromtag`),
+    UNIQUE KEY `from_to` (`fromtag`, `totag`)
+) ENGINE = MyISAM DEFAULT CHARSET = latin1 COLLATE = latin1_german2_ci;
+
+insert into tagsynonyms (fromtag, totag) values ('science fiction', 'sci-fi');
```

Comment:
Does this mean that if I type `science fiction` into the search bar, it should return `sc-fi`? If so, it did not work for me. I also tried it in the opposite direction, and that didn't work either.

Reply:
Changes to `patch-schema-2.sql` require you to `docker compose down` and then `docker compose up` to rebuild the database before testing. It did work for me. Specifically, Zeppelin Adventure is tagged "sci-fi" and not "science fiction", but when I search for "adventure tag:science fiction", Zepplin Adventure is in the results when I search, and not when I do a search on the main branch.

Reply:
Maybe I'm misunderstanding what you were trying to do. I was on the "tags" tab of the search page:

![search tags](https://github.com/user-attachments/assets/58cbb923-8d05-426d-a0c6-8dfff59eae61)

When I type in "science fiction," "sci-fi" does not show up in the results. I deleted the previous codespace and started a new one--would that mean I have the latest version of the database?

Reply:
When I search for `adventure tag:science fiction` on the games tab, I do get Zepplin Adventure in results.

Reply:
I updated the main description of the PR to clarify. I'm trying to make this search <https://ifdb.org/search?searchfor=adventure+tag%3Ascience+fiction> which currently does not return Zeppelin Adventure return Zeppelin Adventure.